src/main/java/com/hms_networks/americas/sc/extensions/historicaldata/HistoricalDataEBDRequest.java

```diff
@@ -0,0 +1,234 @@
+package com.hms_networks.americas.sc.extensions.historicaldata;
+
+import com.hms_networks.americas.sc.extensions.system.time.SCTimeUnit;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Utility methods for building EBD requests for historical data according to Export Block
+ * Descriptor REFERENCE GUIDE RG-0009-00-EN, Publication date 30/08/2023
```

Comment:
Class Javadocs should have an `@author` and `@since` tag. The `@version` tag may also be important and could be added to select classes.

Reply:
changed